daml-lf/archive/src/main/protobuf/com/digitalasset/daml_lf_dev/daml_lf_1.proto

```diff
@@ -512,7 +513,17 @@ enum BuiltinFunction {
 
   TEXT_FROM_CODE_POINTS = 105;  // *Available in versions >= 1.6*
   TEXT_TO_CODE_POINTS = 106; // *Available in versions >= 1.6*
-  // Next id is 131. 130 is GENMAP_SIZE.
+
+  TEXT_TO_UPPER = 131; // *Available in versions >= 1.dev*
+  TEXT_TO_LOWER = 132; // *Available in versions >= 1.dev*
+  TEXT_SLICE = 133;
+  TEXT_SLICE_INDEX = 134; // *Available in versions >= 1.dev*
+  TEXT_CONTAINS_ONLY = 135; // *Available in versions >= 1.dev*
+  TEXT_REPLICATE = 136; // *Available in versions >= 1.dev*
+  TEXT_SPLIT_ON = 137; // *Available in versions >= 1.dev*
+  TEXT_INTERCALATE = 138; // *Available in versions >= 1.dev*
```

Comment:
I would prefer to use an other range of number to add easily stable built-in.

Reply:
I would like also having `experimental` or `unstable` in the name.

Reply:
Both sounds fine to me but I’m not sure the names matter much given that this is only in the `1.dev` proto.

Reply:
I changed the numbers. I'm leaving the names for now at least.